src/yeoman-process-adapter.ts

```diff
@@ -0,0 +1,221 @@
+/*********************************************************************
+* Copyright (c) 2018 Red Hat, Inc.
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which is available at https://www.eclipse.org/legal/epl-2.0/
+*
+* SPDX-License-Identifier: EPL-2.0
+**********************************************************************/
+
+import * as theia from '@theia/plugin';
+import { ChildProcess } from 'child_process';
+
+const path = require('path');
+const fork = require('child_process').fork;
+
+export class YeomanProcessAdapter {
```

Comment:
please comment

Reply:
Done

Reply:
Strange, GitHub does not display the changes but they are present
https://github.com/vitaliy-guliy/theia-yeoman-plugin/blob/separate-thread/src/yeoman-process-adapter.ts